build/Versions.props

```diff
@@ -94,7 +94,8 @@
     <MicrosoftDotNetProjectTemplatesVersion>1.0.0-beta2-20170629-269</MicrosoftDotNetProjectTemplatesVersion>
 
     <!-- Tests -->
-    <MoqVersion>4.7.99</MoqVersion>
+    <MoqVersion>4.9.0</MoqVersion>
+    <XUnitVersion>2.4.0</XUnitVersion>
```

Comment:
Before this, `XUnitVersion` was coming from RepoToolset?  Is this early enough that we don't have a mix of versions anywhere?

Reply:
They respect it coming from Version.props.